pkgs/development/python-modules/crccheck/default.nix

```diff
@@ -1,18 +1,21 @@
-{ lib, stdenv, buildPythonPackage, fetchPypi
+{ lib, stdenv, buildPythonPackage, fetchPypi, isPy3k
 , nose }:
 
-buildPythonPackage rec {
+let
   pname = "crccheck";
-  version = "0.6";
+  version = "1.0";
+in buildPythonPackage {
+  inherit pname version;
 
   buildInputs = [ nose ];
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0ckymm6s5kw08i1j35fy2cfha1hyq94pq1kc66brb552qgjs91jn";
-    extension = "zip";
+    sha256 = "1ay9lgy80j7lklm07iw2wq7giwnv9fbv50mncblqlc39y322vi0p";
   };
 
+  disabled = !isPy3k;
+
   meta = with lib; {
     description = "Python library for CRCs and checksums";
     homepage = "https://bitbucket.org/martin_scharrer/crccheck";
```

Comment:
That link is dead. Pypi mentions https://sourceforge.net/projects/crccheck/. Can you replace it?